modules/programs/floorp.nix

```
@@ -20,7 +20,7 @@ in {
       visible = true;
 
       platforms.linux = {
-        configPath = ".floorp";
+        configPath = ".mozilla";
         vendorPath = ".floorp";
```

I think you meant to replace the vendor path, right?
```suggestion
        configPath = ".floorp";
        vendorPath = ".mozilla";
```

---

Yes ...